src/server.ts

```
@@ -35,6 +36,16 @@ app.post('/', async (request, reply) => {
         reply.code(400).send({error: 'Invalid request'});
         return;
       }
+      
+      if (TX_LIMIT <= 0) {
+        reply.code(400).send({error: `Invalid TX_LIMIT [${TX_LIMIT}]`});
+        return;
+      }
+      let areFeesTooHigh = await feesTooHigh(result.data);
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)